test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -612,4 +619,39 @@ func TestSRIOVNetwork(t *testing.T) {
 	if err := testData.assertPodNetworkStatus(t, clientset, pods, ns, true); err != nil {
 		t.Fatalf("Error when checking the Pod annotation: %v", err)
 	}
+	testData.assertVFName(t, e2eTestData, vfName, pod1, node1, ns)
+}
+
+func (data *testData) assertVFName(t *testing.T, e2eTestData *antreae2e.TestData, vfName, podName, nodeName, ns string) {
+	//  Delete a Pod and check the VF device is recovered with the original interface name.
+	err := e2eTestData.DeletePodAndWait(10*time.Second, podName, ns)
```

Comment:
I feel like the Delete Pod operation should be done by the caller, not as part of this verification function

Reply:
Updated.